docs/README.md

```diff
@@ -0,0 +1,168 @@
+# Roca
+
+BrightScript unit testing.  No device required.
+
+## Getting Started
+Note: All examples use `npm`, but these steps should be easily adapted to [yarn](https://yarnpkg.com/), [pnpm](https://pnpm.js.org/), and many other JavaScript package managers.
+
+### Install
+Roca requires the [brs](https://github.com/sjbarag/brs/) runtime, which runs on [node.js](https://nodejs.org/en/).  If needed, initialize an NPM project with:
```

Comment:
Nit: Seems like there is an extra space before "If needed..."

Reply:
Ow, my heart!  I've got "two spaces after a period" in muscle memory and it's so hard to break!  Seems like this dates back to pre-computer days, and some people were taught to maintain it for monospaced fonts.

Wikipedia article for context: https://en.wikipedia.org/wiki/Sentence_spacing